grove/grovetccfg/grovetccfg.go

```diff
@@ -50,6 +52,9 @@ const UserAgent = "grove-tc-cfg/" + Version
 const TrafficOpsTimeout = time.Second * 90
 const DefaultCertificateDir = "/etc/grove/ssl"
 const GroveConfigPath = "/etc/grove/grove.cfg"
+const ConfigHistory = "cfg_history/"
+const RemapHistory = "remap_history/"
+const GroveProfileType = "GROVE_PROFILE"
```

Comment:
Would you mind moving this to `traffic_control/lib/go-tc/enum.go`? Profile types should be there, so other apps like the TO client can use them.